src/test/resources/add-to-cart.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Adding a product to the cart
+
+  Scenario: Searching for a PS5 and adding it to the cart
+    Given User is on the home page 'https://allegro.pl/'
+    And has cookies accepted
+    When User searches for a PS5
```

Comment:
I believe this step would be reused. It's a good practice to put parameters either between " or '